install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -168,56 +303,92 @@ To enable writing to NFS volumes with SELinux enforcing on each node, run:
 # setsebool -P virt_use_nfs 1
 ----
 
-The `-P` option makes the bool persistent between reboots.
+The `-P` option above makes the bool persistent between reboots.
+
+The *virt_use_nfs* boolean is defined by the *_docker-selinux_* package. If an
+error is seen indicating that this bool is not defined, then ensure that this
+package has been installed.
+
+[[nfs-export-settings]]
+=== Export Settings
 
-Additionally, in order to enable arbitrary container users to read and write the
-volume, each exported volume on the NFS server itself should conform to the
-following:
+In order to enable arbitrary container users to read and write the volume, each
+exported volume on the NFS server should conform to the following:
 
 - Each export must be:
 +
 ----
 /<example_fs> *(rw,root_squash)
 ----
-- The firewall must be configured to allow traffic to the mount point, for NFSv4 the default port is 2049 (nfs).  
-  For NFSv3 there are three ports that need to be configured: 2049 (nfs), 20048 (mountd) and 111 (port mapper).
-+
-NFSv4:
+- The firewall must be configured to allow traffic to the mount point. For NFSv4,
+the default port is 2049 (*nfs*). For NFSv3, there are three ports that need to
+be configured: 2049 (*nfs*), 20048 (*mountd*), and 111 (*portmapper*).
 +
+.NFSv4
 ----
 # iptables -I INPUT 1 -p tcp --dport 2049 -j ACCEPT
 ----
 +
-NFSv3:
-+
+.NFSv3
 ----
 # iptables -I INPUT 1 -p tcp --dport 2049 -j ACCEPT
 # iptables -I INPUT 1 -p tcp --dport 20048 -j ACCEPT
 # iptables -I INPUT 1 -p tcp --dport 111 -j ACCEPT
 ----
+- The NFS export and directory must be set up so that it is accessible by the
+target pods. Either set the export to be owned by the container's primary UID,
+or supply the pod group access using `*suppplementalGroups*`, as shown in
+link:#nfs-supplemental-groups[Group IDs] above. See
+link:pod_security_context.html[Volume Security] for additional pod security
+information as well.
 
-- The NFS export and directory must be set up so that it is accessible by your
-pods. Either set the export to be owned by the container's primary UID, or give
-your pod group based access using `*SuppplementalGroups*`. See
-link:pod_security_context.html[Volume Security] for more information.
+[[nfs-reclaiming-resources]]
+== Reclaiming Resources
+NFS implements the Openshift *Recyclable* plug-in interface. Automatic
+processes handle reclamation tasks based on policies set on each persistent
+volume.
 
+By default, persistent volumes are set to *Retain*. NFS volumes which are set to
+*Recycle* are scrubbed (i.e., `rm -rf` is run on the volume) after being
+released from their claim (i.e, after the user's `*PersistentVolumeClaim*` bound
+to the volume is deleted). Once recycled, the NFS volume can be bound to a new
+claim.
 
-[[nfs-additional-config-and-troubleshooting]]
+[[nfs-automation]]
+== Automation
+As discussed, clusters can be provisioned with persistent storage using NFS in
```

Comment:
The links in the bulleted items do that, but I've just dropped the "As discussed" lead-in.